neurobooth_os/iout/metadator.py

```diff
@@ -52,25 +65,26 @@ def get_tech_obs_logs(conn):
     tech_obs = table_tech_obs_log.query("SELECT * from tech_obs_log")
     return tech_obs
 
-
-def make_new_tech_obs_id():
-    conn = get_conn()
-    tech_obs = get_tech_obs_logs(conn)
-    if list(tech_obs.index) == []:
-        tech_id = "tech_log_1"
-    else:
-        tech_id_last = tech_obs.index[-1]
-        num = int(tech_id_last.split("_")[-1])
-        tech_id = f"tech_log_{num + 1}"
-    return tech_id
-
-
+def make_new_tech_obs_row(conn, subject_id):
+    table = Table("tech_obs_log", conn=conn)
+    cols = [cx for cx in table.column_names if cx!=table.primary_key]
+    vals = [None]*len(cols)
+    sj_ix = cols.index('subject_id')
+    vals[sj_ix] = subject_id
+    return table.insert_rows([tuple(vals)], cols=cols)
```

Comment:
actually you might be able to do this more simply by doing:

```py
cols = [('subject_id')]
vals = ['subject_id']
table.insert_rows(vals, cols)
```
I think postgres should be intelligent enough to figure out that the missing columns are `Null`